datafusion/optimizer/src/analyzer/type_coercion.rs

```diff
@@ -883,14 +883,14 @@ mod test {
     fn scalar_function() -> Result<()> {
         let empty = empty();
         let lit_expr = lit(10i64);
-        let fun: BuiltinScalarFunction = BuiltinScalarFunction::Abs;
+        let fun: BuiltinScalarFunction = BuiltinScalarFunction::Acos;
```

Comment:
```suggestion
        let fun: BuiltinScalarFunction = BuiltinScalarFunction::Abs;
```

Reply:
How about the keep this change?

Reply:
Thank you @Weijun-H ❤️ . 
But I think that the purpose of the test here is not the `abs` function, but rather type coercion (such as Int64->Float64).